### PR TITLE
Implement fee wallet

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

**Comment by robvanmieghem** [comment]
@DylanVerstraete Can you resolve the conflicts?

**Comment by DylanVerstraete** [comment]
> @DylanVerstraete Can you resolve the conflicts?

Yes! :-)

**Review by robvanmieghem — CHANGES_REQUESTED** [review]
Shouldn't  the withdraw fee be substracted  from the amount? and if the amount is less than the withdraw fee, nothing needs to be done or maybe transfer the amount to the feewallet, if not the amount on bsc is not in sync with the amount on the bridgewallet.

File: bsc/bridges/stellar/api/bridge/bridge.go [hunk 1]
```diff
@@ -337,3 +343,19 @@ func (bridge *Bridge) Start(ctx context.Context) error {
...
+
+func (bridge *Bridge) withdraw(ctx context.Context, we WithdrawEvent) (err error) {
+	hash := we.TxHash()
+	log.Info("Creating a withdraw tx", "ethTx", hash)
+	amount := we.amount.Uint64()
+	if amount == 0 {
+		log.Error("Can not withdraw an amount of 0", "ethTx", hash)
+		return
+	}
+	amount -= uint64(WithdrawFee)
```
Comment (DylanVerstraete) [inline bsc/bridges/stellar/api/bridge/bridge.go:355]: What if amount is less than the withdraw fee? I am seeing that this is happening on testnet now. The contract does not forbid the user to withdraw less than 1 TFT. If the amount if lower than the withdraw fee we should transfer this amount to the fee wallet and return

Comment (robvanmieghem) [reply bsc/bridges/stellar/api/bridge/bridge.go:355]: I still had to handle that case

Comment (DylanVerstraete) [reply bsc/bridges/stellar/api/bridge/bridge.go:355]: I am currently working on it, will push a solution

**Pull request merged**